packages/web3-utils/src/types.ts

```diff
@@ -61,4 +73,36 @@ export type JsonEventInterface = {
 	anonymous: boolean;
 };
 
-// need to create Json Contract interface constructor, receive, and fallback
+// TODO need to create Json Contract interface constructor, receive, and fallback
```

Comment:
@nazarhussain I believe this was your TODO

Reply:
Yes that's seems mine.